data_management/tests/initdb.py

```diff
@@ -290,7 +290,8 @@ def init_db(test=True):
     ExternalObject.objects.create(
         updated_by=user,
         data_product=dp_boy_mort_csv,
-        other_unique_name='scottish deaths-involving-coronavirus-covid-19',
+        alternate_identifier='scottish deaths-involving-coronavirus-covid-19',
+        alternate_identifier_type='freetext',
```

Comment:
Sorry I missed this one:
```suggestion
        alternate_identifier_type='Scottish Health and Social Care Open Data id',
```